rules/manager.go

```diff
@@ -439,21 +440,28 @@ func (g *Group) run(ctx context.Context) {
 				}
 				evalTimestamp = evalTimestamp.Add((missed + 1) * g.interval)
 
-				useRuleGroupPostProcessFunc(g, evalTimestamp.Add(-(missed+1)*g.interval))
-
-				iter()
+				g.evalIterationFunc(ctx, g, evalTimestamp)
 			}
 		}
 	}
 }
 
-func useRuleGroupPostProcessFunc(g *Group, lastEvalTimestamp time.Time) {
-	if g.ruleGroupPostProcessFunc != nil {
-		err := g.ruleGroupPostProcessFunc(g, lastEvalTimestamp, g.logger)
-		if err != nil {
-			level.Warn(g.logger).Log("msg", "ruleGroupPostProcessFunc failed", "err", err)
-		}
-	}
+// Default implementation of GroupEvalIterationFunc that is periodically
```

Comment:
Please follow go styleguide: DefaultEvalIterationFunc is ...

Reply:
Apologies. Fixed.